x-pack/plugins/security_solution/public/common/components/event_details/insights/insights.tsx

```diff
@@ -180,7 +180,7 @@ export const Insights = React.memo<Props>(
   }
 );
 
-function hasCorrectAgentTypeAndEventModule(
+export function hasCorrectAgentTypeAndEventModule(
```

Comment:
i just noticed that this is a bug in main, but we should delete this function and instead use the function in x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/investigate_in_resolver.tsx

Reply:
updated to use the new method. I decided against making the change to the current flyout as part of this PR, as I wanted to keep the impact only to the expandable-flyout.
I'm happy to open a separate PR and remove the `hasCorrectAgentTypeAndEventModule` method